### PR TITLE
fix-#10609:zh-translation:docs/tasks/security/authentication/authn-policy/

### PR DESCRIPTION
**Pull request by yanrongshi**
Fixes: https://github.com/istio/istio.io/issues/10609

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by kebe7jun** [comment]
you can push `Fixes: https://github.com/istio/istio.io/issues/10609` in description for issue.

File: content/zh/docs/tasks/security/authentication/authn-policy/index.md [hunk 1]
```diff
@@ -484,7 +484,7 @@ spec:
       notRequestPrincipals: ["*"]
       to:
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/authn-policy/index.md:485]: same indent with `source` I think.
```suggestion
    to:
```

File: content/zh/docs/tasks/security/authentication/authn-policy/index.md [hunk 2]
```diff
@@ -482,9 +482,9 @@ spec:
   - from:
     - source:
       notRequestPrincipals: ["*"]
-      to:
+    to:
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authentication/authn-policy/index.md:485]: https://github.com/istio/istio.io/blob/15cfeb23a6f7758247ecf17bfd3916e7466e45b8/content/en/docs/tasks/security/authentication/authn-policy/index.md#L497-L513

indent still error.

`source` and `to` should have same indents.

**Review by kebe7jun — COMMENTED** [review]
还是不对哦。。

你的编辑器是不是有些自动修正。。

**Comment by yanrongshi** [comment]
> 还是不对哦。
> 
> 你的编辑器是不是非自动修改。。

我重新来一次

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**